src/create-preview-links.ts

```diff
@@ -28,7 +37,11 @@ export const COMMENT_BLOCK_START = '### Preview changes';
  * @param {string} branch - The branch where the theme changes are located.
  * @returns {string} - A JSON string representing the blueprint.
  */
-function createBlueprint(themeSlug: string, branch: string): string {
+function createBlueprint(
+	themeSlug: string,
+	branch: string,
+	repo: string,
+): string {
 	debug(`Creating blueprint for themeSlug: ${themeSlug}, branch: ${branch}`);
```

Comment:
Should we mention the `repo` in this debug line, as we're already mentioning the `themeSlug` and `branch`?

Reply:
Addressed in https://github.com/vcanales/action-wp-playground-pr-preview/pull/24/commits/225fc4c2e147e92ca2c82412889ef2c3286ab6e0